### ISSUE TITLE
Error: Failed to download module, Could not download module "migs" (../../modules/transitivity/main.tf:55

### ISSUE THREAD

**Issue by M88YEH** [terraform-google-modules/terraform-example-foundation#737]
### TL;DR

When running through step 3-networks:step 14 Merge changes to production - The cloudbuild process fails with the following errors:

Error: Failed to download module
Could not download module
"git::https://github.com/terraform-google-modules/terraform-google-vm?ref=v6.6.0":
error downloading
'https://github.com/terraform-google-modules/terraform-google-vm?ref=v6.0.0':

This happens for 3 other modules and only occurs on tf_apply via cloudbuild - The manual tf_apply run locally runs without issue

### Expected behavior

Downloading terraform-google-modules/vm/google 6.0.0 for base_transitivity.migs...
- base_transitivity.migs in .terraform/modules/base_transitivity.migs/modules/mig

module is downloaded with no errors

### Observed behavior

Step #1 - "tf init": Error: Failed to download module
Step #1 - "tf init": 
Step #1 - "tf init": Could not download module "peering" (../../modules/base_shared_vpc/main.tf:84)
Step #1 - "tf init": source code from
Step #1 - "tf init": "git::https://github.com/terraform-google-modules/terraform-google-network?ref=v2.6.0":
Step #1 - "tf init": error downloading
Step #1 - "tf init": 'https://github.com/terraform-google-modules/terraform-google-network?ref=v2.6.0':

Step #1 - "tf init": Error: Failed to download module
Step #1 - "tf init": 
Step #1 - "tf init": Could not download module "migs" (../../modules/transitivity/main.tf:55)
Step #1 - "tf init": source code from
Step #1 - "tf init": "git::https://github.com/terraform-google-modules/terraform-google-vm?ref=v6.0.0":
Step #1 - "tf init": error downloading
Step #1 - "tf init": 'https://github.com/terraform-google-modules/terraform-google-vm?ref=v6.0.0':

Step #1 - "tf init": Error: Failed to download module
Step #1 - "tf init": 
Step #1 - "tf init": Could not download module "templates" (../../modules/transitivity/main.tf:32)
Step #1 - "tf init": source code from
Step #1 - "tf init": "git::https://github.com/terraform-google-modules/terraform-google-vm?ref=v7.6.0":
Step #1 - "tf init": error downloading
Step #1 - "tf init": 'https://github.com/terraform-google-modules/terraform-google-vm?ref=v7.6.0':

Step #1 - "tf init": Error: Failed to download module
Step #1 - "tf init": 
Step #1 - "tf init": Could not download module "access_level_members"
Step #1 - "tf init": (../../modules/restricted_shared_vpc/service_control.tf:33) source code from
Step #1 - "tf init": "git::https://github.com/terraform-google-modules/terraform-google-vpc-service-controls?ref=v2.0.0":
Step #1 - "tf init": error downloading
Step #1 - "tf init": 'https://github.com/terraform-google-modules/terraform-google-vpc-service-controls?ref=v2.0.0':

### Terraform Configuration

```hcl
This issue occurs in cloudbuild so unable to provide terraform configuration
```

### Terraform Version

```sh
git version 2.36.1
Terraform version: 0.13.7  
Go runtime version: go1.14.7
```

**Comment by M88YEH** [comment]
further investigation found this was a local issue
